its/pom.xml

```diff
@@ -61,6 +61,7 @@
           <configuration>
             <systemProperties>
               <maven.home>${maven.home}</maven.home>
+              <JENKINS_VERSION>2.150.3</JENKINS_VERSION>
```

Comment:
Does it have to be upper case? (I'm not used to seeing such...)

Reply:
It shouldnt even be there, I used that to experiment